src/main/groovy/com/lesfurets/jenkins/unit/PipelineTestHelper.groovy

```diff
@@ -680,6 +683,7 @@ class PipelineTestHelper {
         if (exitValue != 0) {
             throw new Exception('Script returned error code: ' + exitValue)
         }
+        return // no default return value requested
```

Comment:
IMHO this is a more accurate mock of the behavior of `sh` when no return is requested